src/rustup-cli/main.rs

```diff
@@ -88,6 +88,7 @@ fn run_rustup() -> Result<()> {
             let opts = self_update::InstallOpts {
                 default_host_triple: TargetTriple::from_host_or_build().to_string(),
                 default_toolchain: "stable".to_string(),
+                profile: String::new(),
```

Comment:
If we're defaulting to `"stable"` should we default to `"default"` for the profile?